Orchestrator/Assets/MemoryAssetTracker.cs

```diff
@@ -159,9 +159,9 @@ T SetDefaults<T>(T orchestrationAsset)
                     return SetDefaults(new OrchestrationImage
                     {
                         S3Location = getImageLocation.Result?.S3, // TODO - error handling
-                        Width = asset.Width,
-                        Height = asset.Height,
-                        MaxUnauthorised = asset.MaxUnauthorised,
+                        Width = asset.Width ?? 0,
+                        Height = asset.Height ?? 0,
+                        MaxUnauthorised = asset.MaxUnauthorised ?? 0,

```

Comment:
`-1`?

Reply:
Yeah I'm not sure what the default should be - the asset from the DB will have a value so something's gone wrong if it doesn't which maybe suggests it should be restricted in some way? Not sure.

Reply:
Yeah true, if in doubt lock it down. `0` makes sense in that case.